src/types/peertube.ts

```diff
@@ -4,11 +4,16 @@ import { v4 } from "uuid";
 
 export default class Peertube extends VM {
   public name = `pt${v4().split("-")[0]}`;
-  public adminEmail = "";
+  public adminEmail = "admin@peertube.com";
```

Comment:
setting default email for admin is dangerous as it can be used as a bad default and user may need it to reset passwords or so